app/src/main/java/app/f3d/F3D/android/Utils/FileType.java

```diff
@@ -0,0 +1,36 @@
+package app.f3d.F3D.android.Utils;
+
+import java.util.Arrays;
+
+public class FileType {
+
+    public final static String[] supportedMimeTypes = {
+            "image/tiff",                    // .tif,
+            "x-world/x-vrml",                // .wrl,
+            "application/dicom",             // .dcm,
+            "application/octet-stream",      // .3ds, .glb, .mhd, .nrrd, .obj, .ply, .pts, .vtk, .vtu, .vtp, .vti, .vtr, .vts, .vtm
```

Comment:
Android is very bad at deducing mimetypes!

Reply:
yeah, wasn't sure on how to proceed as these were the deduced types.
Made a comment on what we can see so that in future can write better code.